cmake/patches/composable_kernel/Fix_Clang_Build.patch

```diff
@@ -71,6 +71,19 @@ index 04674124c..12e8b8b00 100644
 -    LDCONFIG
 -    HEADER_ONLY
 -)
+diff --git a/docs/sphinx/requirements.txt b/docs/sphinx/requirements.txt
```

Comment:
Try to exclude the folder instead. We do not need to spend time on maintaining the dependencies that we do not use. 

Add https://github.com/microsoft/onnxruntime/blob/main/tools/ci_build/github/azure-pipelines/templates/component-governance-component-detection-steps.yml to the end of each job, and adjust the template yaml file as needed.